src/Tribe/Tracker.php

```diff
@@ -33,6 +33,9 @@ public function hook() {
 		// Track the Meta Updates for Meta That came from the correct Post Types
 		add_filter( 'update_post_metadata', array( $this, 'filter_watch_updated_meta' ), PHP_INT_MAX - 1, 5 );
 
+		// Before a meta is removed we mark that is has been modified
+		add_filter( 'delete_post_metadata', array( $this, 'filter_watch_updated_meta' ), PHP_INT_MAX - 1, 5 );
```

Comment:
Move this to use the `deleted_post_meta` action. We don't need the action checks that `filter_watch_updated_meta` does it would be simpler using `filter_watch_added_meta`.